Server/src/models/AuthenticationModel.ts

```diff
@@ -0,0 +1,108 @@
+import { getConnection } from 'typeorm';
+
+import { Accounts } from './entities/Accounts'
+import { ISignUpInformation } from '../genericInterfaces/AuthenticationInterfaces'
+
+/**
+ * This model contains all methods required for obtaining data from the Accounts
+ * table inside the database
+ */
+export class AuthenticationModel {
+
+    /**
+     * Method responsible for completing signup process. Extracting user information
+     * and sets the entity values
+     * @param signUpInfo - User Information from Frontend Request
+     */
+    static async insertSignUpInformation(signUpInfo: ISignUpInformation) {
+
+        let connection = getConnection();
+
+        let signUpInformation = new Accounts();
+        signUpInformation.id;
+        signUpInformation.email = signUpInfo.email;
```

Comment:
because the names match exactly, one option to do this object instantiation could be to use the spread operator:
![image](https://user-images.githubusercontent.com/22376976/102148288-313c6900-3e3a-11eb-8e08-ac777c97678e.png)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
im not 100% sure how to tailor it for the Accounts() and signUpInfo.id

Reply:
Not possible there is no clear mapping